### PR TITLE
npm2deb searches in debian experimental branch for availability of packages.

### PR DESCRIPTION
**Pull request by TusharAgey**
I have changed helper.py in such a way that it fixes the bug:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=852775

### CODE REVIEW

File: npm2deb/helper.py [hunk 1]
```diff
@@ -26,6 +27,10 @@ def search_for_repository(module):
     found = False
     result = {}
     my_print("Looking for existing repositories:")
+    content = subprocess.check_output(["rmadison", "-u", "debian", "node-"+module])
```
Comment (psuhailp) [inline npm2deb/helper.py]: Hi Tushar, 
How about using _getstatusoutput instead of check_output, so that you don't have to import subprocess again.

Comment (TusharAgey) [inline npm2deb/helper.py]: I have made the changes. Now it uses _getstatusoutput instead of check_output!

**Comment by psuhailp** [comment]
Thanks Tushar :+1:

**Comment by TusharAgey** [comment]
Sure! It was fun ;-)

**Pull request merged**